uiowa/uiowa
# Article gallery

Resolves #5524 





# How to test



- `git checkout article-gallery && git pull`
- `blt frontend`
- Sync a site, check an existing article display, and see that there is no gallery image link or any other changes when viewing the page
- Add an article/edit an existing article, and add some gallery images
- Check the display of the page, try the gallery jump link, play with the gallery, etc.

## Additional config checks

Some sites are splitting Articles and need to be tested that 1) the new field appears in the edit form and 2) the gallery and its jump link appear correctly on display and 3) nothing from their respective splits is broken by these additions.

[The following sites may be splitting Articles in some way, and should be tested that nothing breaks or is out of place](https://github.com/uiowa/uiowa/issues/5524#issuecomment-1205370370)

- IISC (check our changes)
- SPPA (check our changes)
- HR (check that no changes are needed)
- Admissions (check that no changes are needed)
- Now (check that no changes are needed)
- others?

(for now.uiowa.edu, the split is not yet enabled in prod, so the following steps are needed to enable its split:
```
blt ds --site=now.uiowa.edu
drush @now.local en now_core
drush @now.local cim --source ../config/sites/now.uiowa.edu --partial
drush @now.local cim
drush @now.local cr
```
)
